templates/map.html

```diff
@@ -1603,7 +1603,7 @@ <h4>{{ i18n('Export Pokémon lists') }}</h4>
       {% if settings.gyms or settings.raids %}
       <div id="gym-stats-tab">
         {% if settings.gyms %}
-        <h6 id="gym-table-title">Gyms</h6>
+        <h6 id="gym-table-title">{{ i18n('Teams') }}</h6>
```

Comment:
In my opinion `Gyms` is better here, because the number after it indicates how many gyms are visible.

Reply:
Oops, went too fast here, that's wasn't a wanted change, I will revert.